benchmark/buffers/buffer-creation.js

```diff
@@ -1,19 +1,22 @@
-SlowBuffer = require('buffer').SlowBuffer;
+'use strict';
+const SlowBuffer = require('buffer').SlowBuffer;
 
-var common = require('../common.js');
-var bench = common.createBenchmark(main, {
-  type: ['fast', 'slow'],
-  len: [10, 1024],
+const common = require('../common.js');
+const bench = common.createBenchmark(main, {
+  type: ['fast-alloc', 'slow-alloc', 'fast-allocraw', 'slow-allocraw'],
```

Comment:
bad naming. `allocraw` is supposed to be `allocUnsafe()`? Personally I still vote for `Buffer.allocRaw()`, but should probably make these names match either way.